test/rest_api/test_storage_service.py

```diff
@@ -586,3 +611,28 @@ def test_range_to_endpoint_map_with_table_param(cql,  rest_api, tablets_enabled)
                 endpoint = entry["value"]
 
                 assert endpoint == expected_endpoint, f"Unexpected endpoint={endpoint} for token_range={token_range}. Expected endpoint was {expected_endpoint}"
+
+@pytest.mark.xfail(reason="rest_api suite doesn't support tablets yet (#17338), run test manually")
```

Comment:
It will just break if we add xfails like that. @bhalevy let's run the API suite with tablets.

Reply:
cc @tchaikov

Reply:
https://github.com/scylladb/scylladb/pull/17474

Reply:
Fixed - introduced usage of the new fixtures.